components/StdHeader.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <nav class="flex bg-white shadow-2xl font-medium h-20 text-lg">
+      <div class="flex min-w-full">
+        
+          <div class="image">
+              <img src="../public/images/top-nav.png" alt="Name Examination" class="h-full">
+          </div>
+
+          <div class="flex gap-24 mt-6 text-blue-800" v-if="this.auth">
+              <a href="" class="ml-16 text-2xl" >Admin</a>
+              <a href="" class="text-2xl">Examine</a>
+              <a href="" class="text-2xl"><nuxt-link to="/search">Search</nuxt-link></a>
+          </div>
+
+          <div class="flex ml-auto" v-if="this.auth">
+
+            <form class="flex items-center">   
+              <label for="allowWordClassificationModal" class="sr-only">NR Number Lookup</label>
+              <div class="relative w-full">
+                  <div class="absolute inset-y-0 left-0 flex items-center pl-3 pointer-events-none">
+                      <svg aria-hidden="true" class="w-7 h-7 text-gray-500 dark:text-gray-400" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M8 4a4 4 0 100 8 4 4 0 000-8zM2 8a6 6 0 1110.89 3.476l4.817 4.817a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 012 8z" clip-rule="evenodd"></path></svg>
```

Comment:
Can a Material Design Icon be used instead of a SVG? Or a local image file? It's kind of ugly to include a SVG in the template.

Reply:
Using a local image file now

Reply:
Looks good, thanks.